clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/store/preference/InAppStore.kt

```
@@ -229,7 +233,11 @@ internal class InAppStore(
         serverSideInApps = if (ssEncryptedInApps.isNullOrBlank()) {
             JSONArray()
         } else {
-            JSONArray(cryptHandler.decrypt(ssEncryptedInApps))
+            try {
+                JSONArray(cryptHandler.decrypt(ssEncryptedInApps))
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)